assets/templates/signup.html

```diff
@@ -38,7 +38,34 @@
       </header>
       <h5>Regístrate</h5>
       <p class="content-subtitle">Crea una cuenta GRATIS hasta 500 Suscriptores.</p>
-
+      <form>
+        <legend>Personal information</legend>
+        <fieldset class="fieldset-small">
+          <label for="name">Nombre:</label>
+          <input type="name" name="lastname" placeholder="Nombre">
+        </fieldset>
```

Comment:
Will we use a fieldset for each field (it sound weird)?

Reply:
Yes, sound weird maybe I can use "field-small" or "holder-small"

Reply:
Lo que quise preguntar es si está bien tener un fieldset (conjunto de campos), para un solo campo.

Reply:
Ah! Si entendí que @GusBaamonde dijo que de esta forma quedan mas ordenados los campos y no hay necesidad de poner un span o un div

Reply:
@GusBaamonde, no digo que esté mal, al final un conjuntyo con un solo elemento (o incluso vacío) es un conjunto igual, pero suena raro. 

A esto te referías al promover el uso de fieldset?

Reply:
Ya revisamos esto, lo subo en el commit